gax-java/gax/src/main/java/com/google/api/gax/tracing/ApiTracer.java

```diff
@@ -108,13 +110,19 @@ default Scope inScope() {
   /** Add an annotation that the attempt was cancelled by the user. */
   default void attemptCancelled() {};
 
+  /** Backport of {@link #attemptFailedDuration(Throwable, java.time.Duration)} */
+  @ObsoleteApi("Use attemptFailedDuration(Throwable, java.time.Duration) instead")
+  default void attemptFailed(Throwable error, org.threeten.bp.Duration delay) {
+    attemptFailedDuration(error, toJavaTimeDuration(delay));
+  };
```

Comment:
Probably can just no-op here. Original implementation wasn't doing anything and this would just  do a time conversion for no reason.

Reply:
Done